WalletWasabi/WabiSabi/Crypto/CredentialIssuer.cs

```diff
@@ -128,7 +135,7 @@ public ICommitableCredentialsResponse PrepareResponse(CredentialsRequest registr
 			// then there is a problem somewhere because this should not be possible.
 			if (Balance + registrationRequest.Delta < 0)
 			{
-				throw new WabiSabiCryptoException(WabiSabiCryptoErrorCode.NegativeBalance);
+				throw new InvalidOperationException("Negative issuer balance");
```

Comment:
even though it's in response to user input, this is not telling the user "you're input is invalid", this is telling the coordinator "my state is invalid", because inflation was detected